src/views/Delegate.vue

```diff
@@ -101,7 +111,54 @@ function clearDelegate(id, delegate) {
   modalOpen.value = true;
 }
 
-const { profiles, addressArray } = useProfiles();
+async function getDelegatesWithScore() {
+  delegatesLoading.value = true;
+  try {
+    const { delegations } = await getDelegatesBySpace(
+      space.value.network,
+      space.value.id
+    );
+
+    const uniqueDelegators = Array.from(
+      new Set(delegations.map(d => d.delegate))
+    ).map(delegate => {
+      return delegations.find(a => a.delegate === delegate);
+    });
+
+    const provider = getProvider(space.value.network);
+    const delegatesAddresses = uniqueDelegators.map(d => d.delegate);
+    const delegationStrategy = extentedSpaces.value
+      .find(s => s.id === form.value.id)
+      .strategies.filter(strategy => strategy.name === 'delegation');
+    const scores = await getScores(
+      space.value.id,
+      delegationStrategy,
+      space.value.network,
+      provider,
+      delegatesAddresses,
+      'latest'
+    );
+
+    uniqueDelegators.forEach(delegate => {
```

Comment:
Can this be done without mutating delegate? I believe we need delegatesWithScore.

Reply:
@gbonumore can you look at this?